app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -43,12 +43,17 @@ class FractionInteractionViewModel private constructor(
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
             pendingAnswerError,
-            answerText.isNotEmpty()
+            true // Allow submit on empty answer
           )
         }
       }
     errorMessage.addOnPropertyChangedCallback(callback)
     isAnswerAvailable.addOnPropertyChangedCallback(callback)
+    // Apply defaults:
+    errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
+      null,
+      true
```

Comment:
Ditto here--please add the method argument names when passing literal values to help make the code more self-documenting.

Reply:
Done